tests/tcp-server/README.md

```diff
@@ -21,8 +21,8 @@ If you make changes to the server:
   local version ```-> imagePullPolicy: Never```
    * Test the changes
  * Include the change as part of the commit that requires the tcp-server change
-   * Build the docker image with an increased version number ```docker build -t tcp-server:v2```
-   * Push the docker image to the public repo
+   * Build the docker image with an increased version number ```docker build -t tcp-server:2.1```
+   * Push the docker image to the public repo ```docker push seanoneillf5/tcp-server:2.1```
```

Comment:
should we not specify the exact repo name and replace it with `<public_repo_name>` as user will be creating their own?

Reply:
https://hub.docker.com/repository/docker/seanoneillf5/tcp-server

I think seanoneillf5 is the repo name?

Reply:
Perhaps we can create a team/company repo?
cc @ciarams87